File: examples/docker-in-e2b-python/template/build_dev.py
from dotenv import load_dotenv
from e2b import Template, default_build_logger
from .template import template

load_dotenv()

if __name__ == "__main__":
    Template.build(
        template,
        alias="e2b-with-docker-dev",
        on_build_logs=default_build_logger(),
    )

File: examples/docker-in-e2b-js/template/build.dev.ts
import { Template, defaultBuildLogger } from "e2b";
import "dotenv/config";
import { template } from "./template";

async function main() {
  await Template.build(template, {
    alias: "e2b-with-docker-dev",
    onBuildLogs: defaultBuildLogger(),
  });
}

main().catch(console.error);
